models/evc.py

```diff
@@ -1523,7 +1661,7 @@ def handle_interface_link_up(self, interface: Interface):
         """
         Handler for interface link_up events
         """
-        if self.archived:  # TODO: Remove when addressing issue #369
+        if self.archived:
```

Comment:
Looks like the `# TODO: Remove when addressing issue #369` got accidentally removed if you could add it again